test/fixtures/VatRegistrationFixture.scala

```diff
@@ -289,6 +289,16 @@ trait VatRegistrationFixture {
        |}
      """.stripMargin).as[JsObject]
 
+  lazy val customerIdentificationBlockWithBPJson: JsObject = Json.parse(
```

Comment:
this won't be reused anywhere else, probably best to keep it in the builder test file